### ISSUE TITLE
Explain the fields in "Font Info & Metadata" section

### ISSUE THREAD

**Issue by probonopd** [fontforge/designwithfontforge.com#202]
http://designwithfontforge.com/en-US/Font_Info_&_Metadata.html contains a section on Font Info & Metadata.

Unfortunately this does not explain what to put where.

It would be worthwhile to have a concise, pragmatic explanation what to put into which field, from today's "industry best practice" perspective. See [here](https://github.com/probonopd/font-leutkirch/issues/11) for some questions that may come to mind when being confronted with this dialog for the first time.

![](http://designwithfontforge.com/en-US/images/fontinfo.png)

**Comment by probonopd** [comment]
If people would like, we could add explanations for the fields based on https://github.com/probonopd/font-leutkirch/issues/11#issuecomment-623149694.

**Comment by ctrlcctrlv** [comment]
I agree with @probonopd. I plan to rewrite this page entirely. While most of this book is really good, I've always thought, this page is very rough. There is far too much focus on basic things people are likely to know, and which are in any event discretionary/to-taste (e.g., the semantics around version numbering&dagger;), yet very important questions are glossed over or ignored entirely:

* What is the difference between sfnt Revision and Version?
  * Why does an sfnt Revision of `2.001` turn into `2.00101`?
* What are the differences between PS Names and TTF Names?
  * Why do PS Names seem to influence TTF Names?
  * Are "TTF Names" even applicable to me if I'm going to be producing a CFF-flavor `.otf` file (that is to say, using cubic B&eacute;ziers in my &laquo;Fore&raquo; layer) or can I ignore them?
  * The page references &ldquo;Preferred Family Name&rdquo; and &ldquo;Preferred Style Name&rdquo;&mdash;yet these strings are nowhere to be found. Where are these set?
* OS/2??
  * Is OS/2 only for IBM OS/2 and its derivatives?
  * Why are weight classes in multiples of 100? Can I put e.g. `150`, and when should I?
  * What is a PFM family? What is an IBM family?
  * How much can I rely on OS/2 DRM?
* What are the WWS Family and Subfamily?
* "Windows does not support font families with more than the 4 basic styles"?? What about InDesign, GIMP, Inkscape, LibreOffice, Krita, and many more?
* If PS UID/XUID are no longer necessary, what is unique then?
* What if I need Unicode?
  * What if my font has different names in different languages, as is common with CJK fonts like ＭＳ 明朝? If my "Name for Humans" can't include Unicode, how do these fonts work?
* How does all this affect CSS?
* What should I put for Weight if this will be a variable font?

(To be clear, I'm not asking that someone else answer these questions before I rewrite it. I know the answers.)

&dagger; It's not even correct about these semantics. "Sadly, the third PATCH version number isn’t available in OpenType font version metadata fields." This is wrong. You can do "Version 1.0;patch 1;FontForge 20200314", and this is relatively common to see. Hack's Version string is, [as of writing](https://github.com/source-foundry/Hack/releases/tag/v3.003), `3.003;[3114f1256]-release; ttfautohint (v1.7) -l 6 -r 50 -G 200 -x 10 -H 181 -D latn -f latn -m "Hack-Regular-TA.txt" -w G -W -t -X ""`

**Comment by probonopd** [comment]
Thanks for taking up this challenge @ctrlcctrlv. As I had to learn in https://github.com/probonopd/font-leutkirch/issues/11, there seems to be much more to this than meets the eye. I hope that thread has some useful material that can be worked into the chapter.
